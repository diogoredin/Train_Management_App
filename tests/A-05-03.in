4
2
123
2
0
0
0
