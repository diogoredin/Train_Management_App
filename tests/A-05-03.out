Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Identificador do serviço: Mostrar serviço com um dado número: Operação inválida: O serviço com o número 123 não existe.
Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Identificador do serviço: Mostrar serviço com um dado número: Operação inválida: O serviço com o número 0 não existe.
Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 